### PR TITLE
Feature 627 coherence spectra

### LINKED ISSUE #627: Add S2S Space-time coherence spectra use-case to replicate PSL transitioned capability
*Replace italics below with details for this issue.*

## Describe the New Feature ##
*Provide a description of the new feature request here.*

### Acceptance Testing ###
*List input data types and sources.*
*Describe tests required for new functionality.*

### Time Estimate ###
*Estimate the amount of work required here.*
*Issues should represent approximately 1 to 3 days of work.*

### Sub-Issues ###
Consider breaking the new feature down into sub-issues.
- [ ] *Add a checkbox for each sub-issue here.*

### Relevant Deadlines ###
*List relevant project deadlines here or state NONE.*

### Funding Source ###
*Define the source of funding and account keys here or state NONE.*

### Assignee ###
- [ ] Select **engineer(s)** or **no engineer** required
- [ ] Select **scientist(s)** or **no scientist** required

### Labels ###
- [ ] Select **component(s)**
- [ ] Select **priority**
- [ ] Select **requestor(s)**

### Projects and Milestone ###
- [ ] Review **projects** and select relevant **Repository** and **Organization** ones or add "alert:NEED PROJECT ASSIGNMENT" label
- [ ] Select **milestone** to next major version milestone or "Future Versions"

## Define Related Issue(s) ##
Consider the impact to the other METplus components.
- [ ] [METplus](https://github.com/dtcenter/METplus/issues/new/choose), [MET](https://github.com/dtcenter/MET/issues/new/choose), [METdatadb](https://github.com/dtcenter/METdatadb/issues/new/choose), [METviewer](https://github.com/dtcenter/METviewer/issues/new/choose), [METexpress](https://github.com/dtcenter/METexpress/issues/new/choose), [METcalcpy](https://github.com/dtcenter/METcalcpy/issues/new/choose), [METplotpy](https://github.com/dtcenter/METplotpy/issues/new/choose)

## New Feature Checklist ##
See the [METplus Workflow](https://dtcenter.github.io/METplus/Contributors_Guide/github_workflow.html) for details.
- [ ] Complete the issue definition above, including the **Time Estimate** and **Funding source**.
- [ ] Fork this repository or create a branch of **develop**.
Branch name: `feature_<Issue Number>_<Description>`
- [ ] Complete the development and test your changes.
- [ ] Add/update log messages for easier debugging.
- [ ] Add/update unit tests.
- [ ] Add/update documentation.
- [ ] Push local changes to GitHub.
- [ ] Submit a pull request to merge into **develop**.
Pull request: `feature <Issue Number> <Description>`
- [ ] Define the pull request metadata, as permissions allow.
Select: **Reviewer(s)**, **Project(s)**, **Milestone**, and **Linked issues**
- [ ] Iterate until the reviewer(s) accept and merge your changes.
- [ ] Delete your fork or branch.
- [ ] Close this issue.

### PR DESCRIPTION
**Pull request by hankenstein2**
## Pull Request Testing ##

- [ ] Describe testing already performed for these changes:</br>

- [ ] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

- [ ] Do these changes include sufficient documentation updates, ensuring that no errors or warnings exist in the build of the documentation? **[Yes or No]**

- [ ] Do these changes include sufficient testing updates? **[Yes or No]**

- [ ] Will this PR result in changes to the test suite? **[Yes or No]**</br>
If **yes**, describe the new output and/or changes to the existing output:</br>

## Pull Request Checklist ##
See the [METplus Workflow](https://dtcenter.github.io/METplus/Contributors_Guide/github_workflow.html) for details.
- [ ] Complete the PR definition above.
- [ ] Ensure the PR title matches the feature or bugfix branch name.
- [ ] Define the PR metadata, as permissions allow.
Select: **Reviewer(s)**, **Project(s)**, and **Milestone**
- [ ] After submitting the PR, select **Linked Issues** with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue and delete your feature or bugfix branch from GitHub.

### CODE REVIEW

File: docs/use_cases/model_applications/s2s/SpaceTimeCrossSpectra.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+# Running METplus
+# ---------------
+#
+# This use case can be run two ways:
+#
+# 1) Passing in hovmoeller_diagram.conf, 
+# then a user-specific system configuration file::
+#
+#        run_metplus.py \
+#        -c /path/to/METplus/parm/use_cases/model_applications/s2s/hovmoeller_diagram.conf \
```
Comment (bikegeek) [inline docs/use_cases/model_applications/s2s/SpaceTimeCrossSpectra.py]: Should this be hovmoeller conf file or the UserScript_obsPrecip_obsOnly_CrossSpectraPlot.conf?

File: docs/use_cases/model_applications/s2s/SpaceTimeCrossSpectra.py [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+#        -c /path/to/user_system.conf
+#
+# 2) Modifying the configurations in parm/metplus_config, then passing in hovmoeller_diagram.conf::
```
Comment (bikegeek) [inline docs/use_cases/model_applications/s2s/SpaceTimeCrossSpectra.py]: same here, should this be hovmoeller?

File: docs/use_cases/model_applications/s2s/UserScript_obsPrecip_obsOnly_CrossSpectraPlot.py [hunk 3]
```diff
@@ -0,0 +1,134 @@
+"""
+UserScript: Make a Hovmoeller plot
```
Comment (bikegeek) [inline docs/use_cases/model_applications/s2s/UserScript_obsPrecip_obsOnly_CrossSpectraPlot.py]: Hovmoeller or Cross spectra plot?

File: docs/use_cases/model_applications/s2s/UserScript_obsPrecip_obsOnly_CrossSpectraPlot.py [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+# There is no python embedding in this use case
+#
+
+##############################################################################
+# Running METplus
+# ---------------
+#
+# This use case can be run two ways:
+#
+# 1) Passing in hovmoeller_diagram.conf,
```
Comment (bikegeek) [inline docs/use_cases/model_applications/s2s/UserScript_obsPrecip_obsOnly_CrossSpectraPlot.py]: hovmoeller or cross spectra conf or UserScript_obsPrecip_obsOnly_CrossSpectraPlot.conf?

**Review by bikegeek — COMMENTED** [review]
Documentation has what appears to be cut-and-paste from hovmoeller plot.  Some instances of hovmoeller need to be replaced with cross spectra references instead.  Or am I mistaken?

**Review by bikegeek — APPROVED** [review]
No more mentions of hovmoeller in the docs.  I approve this request

**Review by georgemccabe — CHANGES_REQUESTED** [review]
We discussed in an email that the cartopy issues in GitHub Actions can be remedied by merging in the latest changes from develop.

Also, the new use case is not in the list of cases to run. This should be added to the following files:

* .github/workflows/main.yml
* internal_tests/use_cases/all_use_cases.txt

Also, make sure to assign a different number to this use case since the pull request for issue #626 (PR #889) is also open and is adding a new use case to the same category. You should be able to skip the number used in #889 and use the next one.

File: docs/use_cases/model_applications/s2s/SpaceTimeCrossSpectra.py [hunk 5]
```diff
@@ -0,0 +1,133 @@
+"""
```
Comment (georgemccabe) [inline docs/use_cases/model_applications/s2s/SpaceTimeCrossSpectra.py]: It looks like this filename should also change to match the use case name.

**Comment by georgemccabe** [comment]
The 2 failed tests on the pull request workflow are from the diff tests (new data doesn't exist in truth data set because it is new) for this use case and #626 (PR #889). This PR can be merged but develop-ref should be updated from develop to include the new test data.

**Pull request merged**